apps/cms/src/hooks/revalidate-page.ts

```diff
@@ -7,60 +7,65 @@ import type { PayloadRequest } from "payload/types";
 // revalidate the page in the background, so the user doesn't have to wait
 // notice that the hook itself is not async and we are not awaiting `revalidate`
 // only revalidate existing docs that are published (not drafts)
-export const revalidatePage =
-  <T extends TypeWithID>(
-    collection: string,
-    getFetchData: (doc: T, req: PayloadRequest) => Promise<unknown>,
-  ): AfterChangeHook<T> =>
-  ({ doc, req, operation }) => {
+export function revalidatePage<T extends TypeWithID>(
+  collectionSlug: string,
+  getFetchData: (doc: T, req: PayloadRequest) => Promise<unknown>,
+): AfterChangeHook<T> {
+  return ({ doc, req, operation }): T => {
     if (
```

Comment:
Add a comment that explains the logic behind this if check

Reply:
Not my code originally, but I think I understand what this is doing. Simplified with a temp variable:
```ts
  const isPublished = "_status" in doc && doc._status === "published";
  if (operation !== "update" || !isPublished) {
    return doc;
  }
```
Better than a comment imo